core/src/main/java/com/github/games647/fastlogin/core/shared/JoinManagement.java

```diff
@@ -138,8 +152,14 @@ private boolean checkPremiumName(S source, String username, StoredProfile profil
     private boolean checkNameChange(S source, String username, Profile profile) {
         //user not exists in the db
         if (core.getConfig().get("nameChangeCheck", false)) {
-            StoredProfile storedProfile = core.getStorage().loadProfile(profile.getId());
+            StoredProfile storedProfile = core.getAuthStorage().loadProfile(profile.getId());
             if (storedProfile != null) {
+                if (storedProfile.isFloodgate()) {
+                    core.getPlugin().getLog()
+                            .warn("Player {} is already stored by FastLogin as a Bedrock Edition player.", username);
```

Comment:
```suggestion
                            .info("Player {} is already stored by FastLogin as a Bedrock Edition player.", username);
```

Sounds more informative than warning the server owner.